infrastructure/terraform/per_workspace/modules/etl/sds/worker/main.tf

```diff
@@ -4,7 +4,7 @@ module "lambda_function" {
 
   function_name = "${var.workspace_prefix}--${var.etl_name}--${var.etl_stage}"
   description   = "${replace(var.workspace_prefix, "_", "-")} ${var.etl_name} (${var.etl_stage}) lambda function"
-  handler       = "etl.sds.worker.${var.etl_stage}.${var.etl_stage}.handler"
+  handler       = "etl.sds.worker.${var.etl_type}.${var.etl_stage}.${var.etl_stage}.handler"
```

Comment:
Just trying to work out what this looks like in my head - what's an example of a handler name with ${var.etl_stage}.${var.etl_stage} twice now?